python/setup.py

```diff
@@ -171,6 +171,17 @@ def get_packages(self):
 # in this directory
 if setup_spec.type == SetupType.RAY:
     setup_spec.extras = {
+        "data": [
+            "dask",
+            "mars",
+            "modin",
+            "pyspark",
+            "tensorflow",
+            "torch",
+            "pandas",
+            "pyarrow==4.0.1",
```

Comment:
I think that we want to allow newer versions of pyarrow, right?
```suggestion
            "pyarrow>=4.0.1",
```